invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/ItemsSearch.js

```diff
@@ -61,14 +70,10 @@ export default class ItemsCheckout extends Component {
         onPaste={e => {
           let queryString = e.clipboardData.getData('Text');
           this.executeSearch(queryString).then(data => {
-            this.checkoutItem(this.props.items[0], this.props.patron).then(
-              () => {
-                this.clearResults();
-                setTimeout(() => {
-                  this.fetchPatronCurrentLoans(this.props.patron);
-                }, 3000);
-              }
-            );
+            if (this.props.items.hits[0].status === 'MISSING') {
```

Comment:
If you have to change the line to:
```
if (invenioConfig.items.available.status.includes(this.props.items.hits[0].status)) {
    this.requestCheckout(this.props.items.hits[0], this.props.patron);
}
```

Because there are multiple "non loanable" states, e.g. MISSING, ON BINDING, etc... but only one "loanable", which for now is LOANABLE